### PR TITLE
Add a 'reset' command to datacats

### PR DESCRIPTION
**Pull request by JackMc**
This command is as described in #170

### CODE REVIEW

**Comment by JackMc** [comment]
Should we make the -s option mandatory (as it is now) or optional, defaulting to 'primary'?

**Comment by JackMc** [comment]
Any opinions on this?

**Comment by wardi** [comment]
optional, defaulting to 'primary' like the other commands

**Comment by JackMc** [comment]
Done

File: datacats/cli/create.py [hunk 1]
```diff
@@ -92,7 +92,32 @@ def create_environment(environment_dir, port, ckan_version, create_skin, site_na
         raise
 
 
-def init(opts):
+def reset(environment, opts):
+    """Resets a site to the default state. This will re-initialize the
+database and recreate the administrator account.
+
+Usage:
+  datacats reset [-s NAME] ENVIRONMENT
```
Comment (wardi) [inline datacats/cli/create.py]: ENVIRONMENT should be optional like the other commands

**Comment by wardi** [comment]
merge now fix ENVIRONMENT command later

**Pull request merged**